build.gradle.kts

```diff
@@ -166,8 +166,7 @@ subprojects {
                     args(
                         it,
                         "--project-directory", projectDir.resolve(cinteropDir),
-                        "--platform", "iOS",
-                        "--cache-builds"
+                        "--platform", "iOS"
```

Comment:
I know it's from upstream, but it should keep cache to speed up builds. I don't know why it was removed